Apps/Admin/Tests/Functional/README.md

```diff
@@ -34,11 +33,11 @@ Create a cypress.env.json and update with passwords or any other environment var
 While creating and debugging tests you will want to run Cypress interactively.
 
 ```bash
-export CYPRESS_BASE_URL=https://dev-admin.healthgateway.gov.bc.ca
+export CYPRESS_BASE_URL=https://mock-admin.healthgateway.gov.bc.ca
 npx cypress open
```

Comment:
NIT:  This can be cleaned up to 
`CYPRESS_BASE_URL=https://mock-admin.healthgateway.gov.bc.ca nix cypress open`

Reply:
I left the lines separate so the command can be copy/pasted without overriding the default environment if desired.